components/organisms/DashboardUserWelcome/DashboardUserWelcome..tsx

```diff
@@ -55,7 +57,14 @@ export const DashboardUserWelcome = () => {
           setisVideoReady(false);
         }
       };
+
+      const resources = async () => {
+        const dataResources = await fetchResourcesData(idUser);
+        console.log(dataResources, "datos en el dashboard");
```

Comment:
Could you please translate this comment to keep the english format?

Reply:
delete line